chingu-voyages/v52-tier3-team-38
# Ross/feature/styling/navbars

# Description
This PR implements a cohesive navigation system across our app based on collaborative team feedback. It unifies the navigation experience across user roles (unauthenticated users, residents, and admins) while improving mobile responsiveness and visual consistency.

## Related to the following Jira/Trello issue 

- Navigation Bar

## Main changes explained:
- Created unified navigation components with consistent styling and branding
- Implemented responsive design patterns for both desktop and mobile views
- Removed unused header components to reduce codebase complexity
- Updated AdminNavbar routes to accurately reflect current app functionality
- Added intuitive icons with proper labeling for improved mobile navigation
- Applied teammate feedback regarding menu structure and mobile display

## How to test:
1. Check into current branch
2. Run npm install and npm run dev
3. Test UserNavbar at any authenticated resident route

Verify horizontal nav items in desktop view
Verify vertical list with icons and labels in mobile view
Confirm Log Out button placement and functionality


4. Test UnauthNavbar at root route

Verify proper display of Services, About Us, and auth buttons
Check mobile responsiveness and icon visibility


5. Test AdminNavbar at /admin

Verify Appointments link leads to correct view
Confirm removal of Profile link
Test mobile menu structure and logout functionality

## Note:
This PR represents a collaborative effort to improve navigation based on team feedback. Key improvements include:

- Simplified mobile menu structure
- Added text labels with icons for better mobile usability
- Consistent styling across all user roles
- Streamlined routing structure for admin views

These changes create a more professional and user-friendly navigation experience while maintaining the app's core functionality.
